shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/BaseRule.g4

```diff
@@ -1326,20 +1369,16 @@ dataTypeLength
     : LP_ NUMBER_ (COMMA_ NUMBER_)? RP_
     ;
 
-characterSet_
-    : (CHARACTER | CHAR) SET EQ_? ignoredIdentifier_
-    ;
-
-collateClause_
-    : COLLATE EQ_? (STRING_ | ignoredIdentifier_)
```

Comment:
@dongzl Why delete this rule？

Reply:
@jingshanglu , This file already has a rule, called `collateClause`. If rename `collateClause_` to `collateClause` directly, it will cause conflict.

I compare `collateClause_` and `collateClause`, merge them into one `collateClause`.

Please check it for me.